docs: Add references to the args and command explanations (#4693)
kubeshop/testkube: docs/docs/test-types/executor-gradle.md

* Default command for this executor: `gradle`
* Default arguments for this executor command: `--no-daemon` `<taskName>` `-p` `<projectDir>`

Parameters in `<>` are calculated at test execution:

> * `<taskName>` - will be `test` in case of tests of type `gradle/test`, `integrationTest` `gradle/integrationTest` on `gradle/integrationTest`, and empty for `gradle/project`

Reviewer: ```suggestion
* `<taskName>` - Will be `test` in case of tests of type `gradle/test`, `integrationTest` `gradle/integrationTest` on `gradle/integrationTest`, and empty for `gradle/project`.
```

Status Test execution completed with success 🥇
```

## Using Different Commands and Arguments

Updating the commands and arguments is possible on both test and execution level.

> As an example, during a debug session, you could pass `pwd` in as the command in order to find out the path you are located at:

Reviewer: ```suggestion
As an example, during a debug session, you could pass `pwd` in as the command in order to find out the current path:
```
